.config/s3.config.php

```diff
@@ -23,6 +23,12 @@
     )
   );
 
+  if (getenv('OBJECTSTORE_S3_SSE_C_KEY_FILE') && file_exists(getenv('OBJECTSTORE_S3_SSE_C_KEY_FILE'))) {
+    $CONFIG['objectstore']['arguments']['sse_c_key'] = trim(file_get_contents(getenv('OBJECTSTORE_S3_SSE_C_KEY_FILE'));
+  } elseif (getenv('OBJECTSTORE_S3_SSE_C_KEY')) {
+    $CONFIG['objectstore']['arguments']['sse_c_key'] = getenv('OBJECTSTORE_S3_SSE_C_KEY');
+  }
+
```

Comment:
IMHO this block should go to the end of the file since `OBJECTSTORE_S3_KEY_FILE` and `OBJECTSTORE_S3_SECRET_FILE` have much more relevance than this.

Reply:
This block is also missing the `else` case...